sw/device/lib/testing/usb_testutils_controlep.c

```diff
@@ -436,3 +437,41 @@ status_t usb_testutils_controlep_init(usb_testutils_controlep_ctx_t *ctctx,
 
   return OK_STATUS();
 }
+
+// Proceed only when the device has been configured; this allows host-side
+// software to establish communication.
+status_t usb_testutils_controlep_config_wait(
+    usb_testutils_controlep_ctx_t *ctxctx, usb_testutils_ctx_t *ctx) {
+  // In simulation the DPI (host) is very responsive, and it will take only
+  // a handful of bus frames to set the configuration; importantly we want
+  // regression simulations to terminate sooner rather than later if there
+  // is a gross connectivity failure.
+  uint32_t kTimeoutUsecs = 8 * 1000;  // 8ms = 8 x 1ms bus frames
```

Comment:
Prefer micros and millis instead.

Reply:
I don't understand that suggestion; are you saying that the SI units 'ms' and 'us' (in the absence of mu) are less clear?